Mathlib/RingTheory/TwoSidedIdeal/Basic.lean

```diff
@@ -169,6 +170,58 @@ instance addCommGroup : AddCommGroup I :=
   Function.Injective.addCommGroup _ Subtype.coe_injective
     rfl (fun _ _ ↦ rfl) (fun _ ↦ rfl) (fun _ _ ↦ rfl) (fun _ _ ↦ rfl) (fun _ _ ↦ rfl)
 
+/-- The coercion into the ring as a `AddMonoidHom` -/
+@[simp]
+def coeAddMonoidHom : I →+ R where
+  toFun := (↑)
+  map_zero' := rfl
+  map_add' _ _ := rfl
+
 end NonUnitalNonAssocRing
 
+section Ring
+
+variable {R : Type*} [Ring R] (I : TwoSidedIdeal R)
+
+instance : SMul R I where smul r x := ⟨r • x.1, I.mul_mem_left _ _ x.2⟩
+
+instance : SMul Rᵐᵒᵖ I where smul r x := ⟨r • x.1, I.mul_mem_right _ _ x.2⟩
+
+instance leftModule : Module R I :=
+  Function.Injective.module _ (coeAddMonoidHom I) Subtype.coe_injective fun _ _ => rfl
```

Comment:
```suggestion
  Function.Injective.module _ (coeAddMonoidHom I) Subtype.coe_injective fun _ _ ↦ rfl
```
I think that's the mathlib style? Similarly line 197

Reply:
changed `=>` to `\mapsto` throughout